### PR TITLE
Kim/story details

### PR DESCRIPTION
**Pull request by kimlarocca**

### CODE REVIEW

File: assets/scss/global.scss [hunk 1]
```diff
@@ -61,6 +61,13 @@ body {
   }
 }
 
+// match the prime vue Divider styles
+hr {
+  border: none;
+  height: 1px;
+  background: rgba(234, 234, 234, 1);
```
Comment (bonomite) [inline assets/scss/global.scss]: use var(--stroke) which will be #1b223c in dark mode... 

if we want the stroke color to be the same in dark mode, use var(--stroke-500)

File: pages/story/[slug].vue [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    </div>
+    <v-image
+      v-if="storyData?.image"
+      :src="storyData?.image.url"
+      :ratio="[1, 1]"
+      :alt="storyData?.image.alt"
+      class="story-page-image mb-2"
+    />
+    <section>
+      <p class="story-page-date mb-3">
```
Comment (bonomite) [inline pages/story/[slug].vue]: we can use the PipeData.vue component here

File: pages/story/[slug].vue [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<script setup>
+import VImage from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VImage.vue'
+
+// TO DO - replace dummy data with BFF data
+import storyData from './story-data.json'
+
+// format the date to show X min ago, X hours ago, or the date in Month Day, YYYY format
+const formatDate = (date) => {
```
Comment (bonomite) [inline pages/story/[slug].vue]: not need when using the PipeData.vue component below that already uses the howLongAgo helper function

File: pages/story/[slug].vue [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    return '1 hour ago'
+  } else {
+    return `${hours} hours ago`
+  }
+}
+</script>
+
+<template>
+  <div v-if="storyData" class="story-page">
+    <div class="p-3 flex align-items-center">
```
Comment (bonomite) [inline pages/story/[slug].vue]: replace with 

const backHome = () => {
 // gtag tracking here
 navigateTo('/home') 
 }
<Button label="Home" icon="pi pi-chevron-left" @click="() => {backHome}" />

Comment (bonomite) [reply pages/story/[slug].vue]: you can delete the arrow svg file as well

File: pages/story/[slug].vue [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    <section>
+      <p class="story-page-date mb-3">
+        <nuxt-link :to="storyData.source_url" class="no-underline">
+          {{ storyData.source }}
+        </nuxt-link>
+        <span class="mx-1">|</span>
+        {{ formatDate(storyData.updated_date) }}
+      </p>
+      <h1 class="mb-1">{{ storyData.title }}</h1>
+      <p class="story-page-author mb-4">
```
Comment (bonomite) [inline pages/story/[slug].vue]: replace with VByline.vue component from the main-v2 branch of the DS

**Review by bonomite — CHANGES_REQUESTED** [review]
a few comments... let me know if you need any help with any of it. Thanks!

File: pages/story/[slug].vue (hunk 5 above)
Comment (kimlarocca) [reply pages/story/[slug].vue]: done!

File: assets/scss/global.scss (hunk 1 above)
Comment (kimlarocca) [reply assets/scss/global.scss]: ok!

File: pages/story/[slug].vue (hunk 2 above)
Comment (kimlarocca) [reply pages/story/[slug].vue]: got it!

**Review by bonomite — APPROVED** [review]
good for now. I made a few tiny updates on the branch...

**Pull request merged**